server/src/routes/api/v1/squidsRouter.js

```diff
@@ -1,23 +1,37 @@
 import express from "express";
 
 import { Squid } from "../../../models/index.js";
+import { cleanUserInput } from "../../../services/cleanUserInput.js";
 import { nextWrapper } from "../../lib/nextWrapper.js";
 
 export const squidsRouter = new express.Router();
 
 squidsRouter.get(
   "/",
   nextWrapper(async (req, res) => {
-    const offset = req.query.offset || 0
-    const limit = req.query.limit || 10
-    
-    const squidsQuery = Squid.query();
+    const specialPowers = Squid.jsonSchema.properties.specialPower.enum;
```

Comment:
oh cool! I hadn't even thought about calling on jsonSchema to grab these values 🙂 To shorten this up a bit, and anticipating future use, you might consider defining 

```
static get specialPowersOptions() {
   return ["....
}
```

on your model and then use that array both here and in your jsonSchema